lib/inertia_rails/controller.rb

```diff
@@ -6,8 +6,10 @@ module Controller
 
     module ClassMethods
       def inertia_share(**args, &block)
-        InertiaRails.share(args) if args
-        InertiaRails.share_block(block) if block
+        before_action do
```

Comment:
because we clear data in the middleware, we need to re-share the data on each request